assets/sass/_content-post.scss

```diff
@@ -33,6 +33,56 @@
   }
 }
 
+/* tutorial image */
+
+  .img-right  {
```

Comment:
These seem duplicates of some classes we already have, e.g. the ones used here: https://ampproject-custom.firebaseapp.com/docs/guides/pwa-amp/amp-as-pwa. Can you merge/reuse the helper classes?